plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -93,6 +93,8 @@ func init() {
 	factory.RegisterFitPredicate("HostName", predicates.PodFitsHost)
 	// Fit is determined by node selector query.
 	factory.RegisterFitPredicate("MatchNodeSelector", predicates.PodSelectorMatches)
+	// Optional, cluster-autoscaler friendly priority function - give used nodes higher priority.
+	factory.RegisterPriorityFunction("MostRequestedPriority", priorities.MostRequestedPriority, 1)
```

Comment:
It seems like this will fight with LeastRequestedPriority.

This would probably be a good time for us to re-think the mechanism we use to configure the scheduler policies Today we have a scheduler config file if the user wants to override the defaults, but really we should be using ConfigMap (see #25497). We need some mechanism that works when the user can't access the machine where the scheduler is running, like on GKE.